lib/samson/hooks.rb

```diff
@@ -23,7 +23,8 @@ class UserError < StandardError
       :after_deploy,
       :before_docker_build,
       :after_docker_build,
-      :after_job_execution
+      :after_job_execution,
+      :additional_exports
```

Comment:
Rather than `:additional_exports`, would you rename the hook to something like `:job_env_vars`.  I'd like to have `job` in the name, so it's clear it's related to job execution, and that makes it clear to me that it involves the environment variables you set in the job's process.